src/index.css

```diff
@@ -8,5 +8,51 @@
   font-family: 'Work Sans', sans-serif;
 }
 
+/* Global Styles */ 
 
+/* Ensures a parent element with this class name stretches to the height of floated children */
+.clearfix:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+}
+
+.primarySection {
+  padding: 120px 0;
+}
+
+.background2 {
+  background-color: gray;
```

Comment:
Doesn't matter much, but fwiw, gray is the American spelling. Grey is the British spelling.